src/umpire/ResourceManager.cpp

```diff
@@ -268,6 +272,16 @@ std::vector<std::string> ResourceManager::getResourceNames()
   return registry.getResourceNames();
 }
 
+std::vector<std::string> ResourceManager::getSharedAllocNames()
```

Comment:
```suggestion
std::vector<std::string> ResourceManager::getSharedAllocatorNames()
```

I don't think we need to save 4 characters 😄

Reply:
I vote for removing the vowels from all names.... 🤣

Reply:
Done!